README.md

```diff
@@ -2,6 +2,15 @@
 
 This plugin helps to generate server and client certificates from an existing CA in a secure manner. The CA Key is never exposed to the remote host. The generation of the certicates happen on the localhost.
 
+## Features
+
+- lives completely in ansible vault
```

Comment:
This depends on actual usage. It is recommended to store some/all your config + keys in vault but it is not enforced by the module.